Gopkg.toml

```diff
@@ -8,7 +8,7 @@
 
 [[constraint]]
   name = "github.com/tendermint/tendermint"
-  version = "v0.27.0-dev1"
+  version = "v0.30.0"
```

Comment:
why iavl depends on Tendermint again?

Reply:
Because it uses some parts of what previously tmlibs: https://github.com/tendermint/iavl/commit/c8114f186987fa8c0489afd00ae40846eee9fb34#diff-836546cc53507f6b2d581088903b1785
more dependencies introduced due to:
https://github.com/tendermint/iavl/pull/105 (https://github.com/tendermint/tendermint/pull/1912, https://github.com/tendermint/tendermint/pull/2298)

Reply:
also ref: https://github.com/tendermint/iavl/issues/46